templates/modcontrol/auditlog.html

```diff
@@ -0,0 +1,48 @@
+$def with (log, form={})
+$:{RENDER("common/stage_title.html", ["Audit Log", "Moderator Control Panel", "/modcontrol"])}
+
+<div class="content audit-log">
+  <table>
+    <thead>
+      <tr>
+        <th width="15">User</th>
+        <th width="15">Staff member</th>
+        <th width="40">Change</th>
+        <th width="30">Time</th>
+      </tr>
+    </thead>
+    <tbody>
+      $for event in log:
+        <tr>
+          <td>
+            <a href="/~$event['user_name']">$event['user_name']</a> &middot; <a href="/modcontrol/auditlog?username=$event['user_name']">All</a>
+          </td>
+          <td>
+            <a href="/~$event['mod_name']">$event['mod_name']</a> &middot; <a href="/modcontrol/auditlog?staff=$event['mod_name']">All</a>
+          </td>
+          <td>$:event['content']</td>
+          <td>${DATE(event['unixtime'])} at ${TIME(event['unixtime'])}</td>
+        </tr>
+    </tbody>
+  </table>
+
+  <div class="form skinny">
+    <form method="GET" action="/modcontrol/auditlog">
+      <label for="username">Username</label>
+      <input type="text" name="username" class="input" id="username" value="$form['username']">
```

Comment:
There’s already an element with the id “username” in the header. For now, just picking a non-colliding id is probably fine; I’ll change it to use `<span class="label">` later on.